source/getting-started/core-concepts.md

```diff
@@ -1,58 +1,136 @@
-To get started with Ember.js, there are a few core concepts you
-should understand.
+Before moving to more focused topics, it is important to review several
+high-level Ember.js concepts.
 
 ## Templates
 
-Templates, written in the Handlebars language, describe the user interface of
-your application. In addition to plain HTML, templates can contain expressions,
-like `{{title}}` or `{{author}}`, which take information from a component or
-controller and put it into HTML. They can also contain helpers, like
-`{{if isAdmin}}30 people have viewed your blog today.{{/if}}.` Finally, they can
-contain components, such as if a template listing out all the blog posts in an
-app rendered a component for each post.
+Ember.js uses templates to organize the layout of HTML in an application. Most
```

Comment:
I'm not sure this will ever be resolved. It is kind of like "Matthew" and "Matt". To people I just meet, I am "Matthew". To those who know me, "Matt". I don't think it is too awkward to migrate from "Ember.js" to "Ember" after using the former a few times.

Open to ideas, but I don't suggest over-analyzing it.